server/core/app.ts

```diff
@@ -33,78 +34,50 @@ export const returnExpress: (
       >,
       { moduleName, addModule }: UIServerModule
     ) => {
-      log(
-        serverName,
-        'startup',
-        'core',
-        'moduleMount',
-        `Mounting module '${moduleName}'`
-      );
+      logger.info(`Mounting module '${moduleName}'`);
       const config = getConfig();
       const { mountPoint, routerForModule } = addModule(
-        (moduleName) => generateLoggers(serverName, moduleName, 'startup'),
+        (moduleName) => generateLogger(moduleName),
```

Comment:
With the new logger here (and the fact we know the moduleName, server name etc), rather than pass the generator, could we pass the result of generateLogger's entry ( as we know the function name/params)? Addresses a concern @nictownsend had re the passing of the generator to all modules

Reply:
Sure, will update.